gordo_components/builder/build_model.py

```diff
@@ -61,6 +67,8 @@ def build_model(
         Mapping of the Dataset to initialize, following the same logic as model_config.
     metadata: dict
         Mapping of arbitrary metadata data.
+    azureml_config: mlflow.tracking.MlflowClient
+        Client with tracking uri set to AzureML.
```

Comment:
Not actually a parameter, and I would strongly urge against making it as such. Current parameters here are suppose to represent basically individual keys per machine, where the contents of each are quite literally the loaded yaml's dump of each corresponding key.